Application/EdFi.Ods.Api/Attributes/NoDangerousTextAttribute.cs

```diff
@@ -22,7 +22,8 @@ protected override ValidationResult IsValid(object value, ValidationContext vali
             }
 
             return CrossSiteScriptingValidation.IsDangerousString(s, out int matchIndex)
-                ? new ValidationResult($"{validationContext.DisplayName} contains a potentially dangerous value.")
+                ? new ValidationResult($"{validationContext.DisplayName} contains a value that could be dangerous for downstream systems using this data. Try to avoid the use of special symbols like '<', '>' or '&' without surrounding spaces.", 
```

Comment:
Data validation errors are to target end-users of the source applications (e.g. SIS software). Making this message more user-friendly.